src/munify.ml

```diff
@@ -1168,6 +1173,10 @@ module struct
         log_eq env "Rigid-Same" conv_t c c' (dbg, sigma0) &&=
         use_evar_conv env c c')
 
+    | Proj (c1, t1), Proj (c2, t2) when Names.Projection.equal c1 c2 ->
+      report (
+        log_eq env "Proj-Same" conv_t c c' (dbg, sigma0) &&=
```

Comment:
Probably not good to put a random name here. I can change that, if necessary.

Reply:
"Proj-Same" sounds right to me.